### PR TITLE
WRP-19325: Updates node modules' major version for ui-test-utils.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] I have run automated testing and it is passed
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
I updated chalk from version 4.1.2 to 5.3.0.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Chalk is ESM module now: 
- https://github.com/chalk/chalk/releases/tag/v5.0.0

This means that we need to use dynamic imports to access those packages or change our repo to be ESM also. the first option was chosen.

### Links
[//]: # (Related issues, references)
WRP-19325

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by hong6316 — COMMENTED** [review]
Could you modify `ui/wdio.conf.js` like `src/build-apps.js` style and test again?

1. add `let chalk;` on the top
2.
	onPrepare: function () {
		return import('chalk').then(({default: _chalk}) => {
			chalk = _chalk
			buildApps('ui')
		});
	},`
3. revert `afterTest` and `afterSuite`

File: ui/wdio.conf.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 const buildApps = require('../src/build-apps'),
-	fs = require('fs'),
-	chalk = require('chalk');
+	fs = require('fs');
```
Comment (hong6316) [inline ui/wdio.conf.js:2]:
```suggestion
	fs = require('fs');
let chalk;
import('chalk').then(({default: _chalk}) => {
	chalk = _chalk
});
```

File: ui/wdio.conf.js [hunk 2]
```diff
@@ -27,30 +26,34 @@ exports.config = configure({
...
-		const filename = encodeURIComponent(testCase.title.replace(/\s+/g, '-'));
-		// build file path
-		const filePath = this.screenshotPath + filename + '.png';
-		if (!fs.existsSync(this.screenshotPath)) {
-			fs.mkdirSync(this.screenshotPath, {recursive: true});	// May only work recursively on Node 10.12+
-		}
-		// save screenshot
-		browser.saveScreenshot(filePath);
-		console.log('\n\tScreenshot location:', filePath, '\n');
+		import('chalk').then(({default: chalk}) => {
```
Comment (hong6316) [inline ui/wdio.conf.js:29]: revert it

File: ui/wdio.conf.js [hunk 3]
```diff
@@ -27,30 +26,34 @@ exports.config = configure({
...
+			browser.saveScreenshot(filePath);
+			console.log('\n\tScreenshot location:', filePath, '\n');
+		});
 	},
 	afterSuite: function (_suite) {
 		// Note: This duration will be less than reported by the various reporters. This seems like
 		// the best we can do without access to the internal runner
 		if (this.__duration > 80000) {
-			console.log(chalk.yellow(`Long running suite: ${_suite.title}: ${this.__duration}ms`));
+			import('chalk').then(({default: chalk}) => {
```
Comment (hong6316) [inline ui/wdio.conf.js:54]: revert it

**Review by hong6316 — COMMENTED** [review]
Could you modify `ui/wdio.conf.js` like `src/build-apps.js` style and test again?

1. add these lines on the top
> let chalk;
> import('chalk').then(({default: _chalk}) => {
> 	chalk = _chalk
> });
2. revert `afterTest` and `afterSuite`

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**